src/file-3.xoo

```diff
@@ -0,0 +1,3 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:34572/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:34572/project/issues?id=sample&branch=feature1&issues=AWD5cwTPotkWn4IrdchZ&open=AWD5cwTPotkWn4IrdchZ)

[ISSUE_KEY]: AWD5cwTPotkWn4IrdchZ